README.md

```diff
@@ -0,0 +1,25 @@
+# Picoprobe
+
+A Picoprobe is a Raspberry Pi RP2040-based Single-Wire Debug (SWB) adaptor. You can make one from a Raspberry Pi Pico — hence the name — or almost any third-party RP2040-based board. Use it to connect your RP2040-based application board to your personal computer for debugging through `gdb`, directly or via an IDE such as Microsoft Visual Studio Code.
+
+The original version of this code only supported the Pico. This update adds support for three more boards, each of which provides a [STEMMA QT connector](https://learn.adafruit.com/introducing-adafruit-stemma-qt/what-is-stemma-qt)/[Quicc](https://www.sparkfun.com/qwiic) for easy cabling to the application board under test. These connectors are intended for use with I&sup2;C devices, but here we’re using their pins for GPIO and power.
+
+The code also uses the WS2812 RGB LED on those boards that have one as an SWD activity indicator.
+
+If your chosen board is not one of those so far supported by the Picoprobe code, you should be able to use with just a few small code changes.
+
+## Build the Picoprobe code
+
+To build for your own Picoprobe:
+
+1. Edit the `CmakeLists.txt` file and change the value of  `PICO_BOARD` in line 16.
```

Comment:
Line number change over time. Better to reference this by name and not by line number. Less work later.